pkgs/development/libraries/tpm2-tss/default.nix

```diff
@@ -27,14 +27,40 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  postPatch = "patchShebangs script";
+  patches = [
+    # Do not rely on dynamic loader path
+    # TCTI loader relies on dlopen(), this patch prefixes all calls with the output directory
+    ./no-dynamic-loader-path.patch
+  ];
+  postPatch = ''
+    patchShebangs script
+    substituteInPlace src/tss2-tcti/tctildr-dl.c --replace \
+      '@PREFIX@' \
+      $out/lib/
+    substituteInPlace ./test/unit/tctildr-dl.c --replace \
+      ', "libtss2' \
+      ", \"$out/lib/libtss2"
+    substituteInPlace ./test/unit/tctildr-dl.c --replace \
+      ', "foo' \
+      ", \"$out/lib/foo"
+    substituteInPlace ./test/unit/tctildr-dl.c --replace \
+      ', TEST_TCTI_NAME' \
+      ", \"$out/lib/\"TEST_TCTI_NAME"
+  '';
 
   configureFlags = [
     "--enable-unit"
     "--enable-integration"
   ];
 
   doCheck = true;
+  preCheck = ''
+    # Since we rewrote the load path in the dynamic loader for the TCTI
+    # The various tcti implementation should be placed in their target directory
+    # before we could run tests
+    installPhase
+    dontInstall=1 # install already done, dont need another one
```

Comment:
```suggestion
    # install already done, dont need another one
    export dontInstall=1
```

Reply:
The export is not needed as far as I can tell